src/middlewares/validationResults.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response, NextFunction } from 'express'
+import { ClassConstructor, plainToClass } from 'class-transformer'
+import { validate } from 'class-validator'
+import logger from '../lib/logger'
+import { ValidationError } from '../errors/HttpErrors'
+import { CreateTaskDto, ReplaceTaskDto, UpdateTaskDto, IdParam, GetDeleteTaskDto } from '../dto/task.dto'
+
+type SourceData = 'body' | 'query' | 'params'
+type TaskDto = CreateTaskDto | UpdateTaskDto | ReplaceTaskDto | ReplaceTaskDto | IdParam | GetDeleteTaskDto
+
+export default function validateRequest (input: ClassConstructor<TaskDto>, source: SourceData) {
```

Comment:
`ClassConstructor<any>` it's fine in this case have `any` type

Reply:
fixed d05ad7787eac1bd92d123586f034466887fdb451